tests/base.py

```diff
@@ -244,7 +250,7 @@ def run_and_verify_check_mode(self, conn_id):
 
         found_catalog_names = set(map(lambda c: c['stream_name'], found_catalogs))
         print(found_catalog_names)
-        self.assertSetEqual(self.expected_streams(), found_catalog_names, msg="discovered schemas do not match")
+        self.assertGreater(len(self.expected_streams()), len(found_catalog_names), msg="discovered schemas do not match")
```

Comment:
why was this condition changed ?

Reply:
Since, the total streams available may not be same as the streams for which discovery is going to run.
I have a better condition for this, which I'll add in my next commit.

Reply:
I've changed the condition from `assertGreater` to `assertTrue` with `issubset` condition.